docs/ui/components/Table/Cell.tsx

```diff
@@ -7,8 +7,12 @@ type CellProps = PropsWithChildren<{
   textAlign?: TextAlign;
 }>;
 
-export const Cell = ({ children, textAlign }: CellProps) => (
-  <td css={css({ borderBottom: 0, verticalAlign: 'middle', wordBreak: 'break-word', textAlign })}>
-    {children}
-  </td>
+export const Cell = ({ children, textAlign = TextAlign.Left }: CellProps) => (
```

Comment:
Had to add this one here explicitly, bc of our global styling. Don't want to edit that yet, because that will affect the current docs